parts/k8s/kuberneteswindowssetup.ps1

```diff
@@ -188,11 +186,13 @@ Write-AzureConfig()
     "aadClientSecret": "$AADClientSecret",
     "resourceGroup": "$global:ResourceGroup",
     "location": "$Location",
+    "vmType": "$global:VmType",
     "subnetName": "$global:SubnetName",
     "securityGroupName": "$global:SecurityGroupName",
     "vnetName": "$global:VNetName",
     "routeTableName": "$global:RouteTableName",
     "primaryAvailabilitySetName": "$global:PrimaryAvailabilitySetName",
+    "primaryScaleSetName": "$global:PrimaryScaleSetName",

```

Comment:
where is primaryScaleSetName used?

Reply:
Both `vmType` and `primaryScaleSetName` are part of the cloud provider config for Azure for v1.10.